internal/app/admin/http/handler.go

```diff
@@ -126,11 +138,29 @@ func resourceToString(resource cache.Resource) (string, error) {
 	return stringify.InterfaceToString(resourceString)
 }
 
-func getCacheKeyParam(path string) (string, error) {
-	// Assumes that the URL is of the format `address/cache/parameter` and returns `parameter`.
+func getParam(path string) (string, error) {
+	// Assumes that the URL is of the format `address/endpoint/parameter` and returns `parameter`.
 	splitPath := strings.SplitN(path, "/", 3)
 	if len(splitPath) == 3 {
 		return splitPath[2], nil
 	}
-	return "", fmt.Errorf("unable to parse cache key from path: %s", path)
+	return "", fmt.Errorf("unable to parse parameter from path: %s", path)
+}
+
+func logLevelHandler(l log.Logger) http.HandlerFunc {
+	return func(w http.ResponseWriter, req *http.Request) {
+		if req.Method == "POST" {
+			logLevel, _ := getParam(req.URL.Path)
```

Comment:
If this returns an error I think we should exit. `zap.ParseLogLevel` will default to Info IIRC.  May produce unexpected behavior.

Reply:
I think that's what it currently does? Returns a BadRequest error to the client

Reply:
we don't assign the error in this line though. If we can't parse the param in `getParam`, `zap.ParseLogLevel("")` is effectively called, which defaults to the info log level, no?

Reply:
oh, I was looking at the wrong line. Yeah we should catch that error I don't know why I didn't include it.

Reply:
actually nvm, I think this is me misreading the implementation for `zap.ParseLogLevel`.